docs/source/370_jupyter_lmod.rst

```diff
@@ -0,0 +1,56 @@
+Jupyter and Lmod
+================
+
+It is possible to use Lmod inside of Jupyter with the terminal or jupyter-lmod.
+
+jupyter-lmod is a frontend and server extension for Jupyter (notebook and jupyterlab).
+The extension provides most of Lmod functionalities graphically:
+    - Browser and filter available modules
+    - Load and unload modules
+    - Display a module content
+    - Create and load collections
+
+Installation
+~~~~~~~~~~~~
+
+To install jupyter-lmod extension: ::
+
+    pip install jupyterlmod
```

Comment:
This is not exactly true as we speak, but it is possible. I just need to fix jupyter-lmod setup.py to install the jupyterlab extension. I'll fix it before changing this PR from draft to mergeable.

Reply:
Fixed with the release of jupyter-lmod 4.0.0.